gapic/src/main/com/google/gapid/models/Devices.java

```diff
@@ -24,6 +24,7 @@
 import com.google.gapid.proto.device.Device;
 import com.google.gapid.proto.device.Device.Instance;
 import com.google.gapid.proto.service.Service;
+import com.google.gapid.proto.service.Service.ValidateDeviceResponse;
```

Comment:
Please don't import this class, instead refer to it via `Service.ValidateDeviceResponse`. This makes it easier to know what proto file the message came from.

Reply:
Done.